src/CMakeLists.txt

```diff
@@ -85,6 +85,12 @@ else()
   set(TOIT_NETWORK_LIBS mbedtls)
 endif()
 
+if (CMAKEISDUMB${CMAKE_SYSTEM_NAME} MATCHES "CMAKEISDUMBMSYS")
```

Comment:
Even though "MSYS" is in quotes, Cmake will expand it because it's also a variable (apparently).  Putting a text in front prevents this.

Reply:
It looks like you can just change the CMP:
https://cmake.org/cmake/help/latest/policy/CMP0054.html#policy:CMP0054

`cmake_policy(SET CMP0054 NEW)`